CHANGELOG.md

```diff
@@ -23,6 +24,9 @@
   `canonicalize_address`/`humanize_address`.
 - Add `cosmwasm_vm::testing::test_io` for basic memory allocation/deallocation
   testing between host and guest.
+- Make `ValidationErr.msg` a dynamic `String` including relevant runtime
```

Comment:
Thanks for updating changelog, something I am poor on doing

Reply:
It has been "pre-release" so long, I figured no one cares about changes. But yeah, from 0.6 on, we should track the changes.